Library/Formula/ucommon.rb

```diff
@@ -14,16 +12,24 @@ class Ucommon < Formula
   depends_on "pkg-config" => :build
   depends_on "gnutls"
 
+  needs :cxx11
+
+  fails_with :clang
```

Comment:
@mikemcquaid Yes, that was just a test, I need to edit the formula with a comment pointing to the bug report.